src/rgw/rgw_sal_motr.cc

```diff
@@ -1866,6 +1869,7 @@ int MotrObject::read_mobj(const DoutPrefixProvider* dpp, int64_t off, int64_t en
   //
   // TODO: We leave proper handling of offset in the future.
   bs = this->get_optimal_bs(end - off);
+  block_start_off = bs;
```

Comment:
`block_start_off` is actually starting offset of the next block so should be renamed as `next_block_off` to avoid any confusion.

Reply:
No need to rename after correcting the logic of calculating the block_start_off